packages/ui-components/src/Shared.styles.ts

```diff
@@ -9,13 +9,16 @@ import SUIDropdown from 'semantic-ui-react/dist/commonjs/modules/Dropdown';
 import SUIInput from 'semantic-ui-react/dist/commonjs/elements/Input';
 
 import { FONT_SIZES, MARGIN_SIZES } from './constants';
-import { HeaderProps, DynamicSizeTextProps, StackProps, StyledNavLinkProps, SubHeaderProps, WithSpaceAroundProps } from './StyleProps';
+import { DynamicSizeTextProps, HeaderProps, FlexItemProps, StackProps, StyledNavLinkProps, SubHeaderProps, WithSpaceAroundProps, WrapperDivProps } from './StyleProps';
 
 // FIXME: customize as needed
 export const Dropdown = styled<any>(SUIDropdown)`
   color: ${props => props.theme.black};
 `;
-export const Input = styled<any>(SUIInput)``;
+
+export const Input = styled<any>(SUIInput)`
```

Comment:
Ah, I think you need to remove the `export {Input} from 'semantic-ui-react'` in index.ts, or else there'll be 2 of them